src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -100,15 +99,7 @@ private static void setupLogger(Application application) {
 	}
 
 	private static void setMouseDrawer(Application application) {
-		application.getFreePanel().addMouseListener(new MouseInputAdapter() {
-			@Override
-			public void mouseClicked(java.awt.event.MouseEvent e) {
-				int correctedX = e.getX() - 268;
-				int correctedY = e.getY() - 227;
-				
-				DriverFeature.getDriverManager().getCurrentDriver().operateTo(correctedX, correctedY);
-			};
-		});
+		application.getFreePanel().addMouseListener(new MouseListener(application.getFreePanel()));
```

Comment:
amazing!!! Please make it simpler. Try to remove application.getFreePanel() code duplication.